packages/postgres-database/src/simcore_postgres_database/models/projects_comments.py

```diff
@@ -0,0 +1,55 @@
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+from .base import metadata
+from .projects import projects
+
+projects_comments = sa.Table(
+    "projects_comments",
+    metadata,
+    sa.Column(
+        "comment_id",
+        sa.BigInteger,
+        nullable=False,
+        autoincrement=True,
+        primary_key=True,
+        doc="Primary key, identifies the comment",
+    ),
+    sa.Column(
+        "project_uuid",
+        sa.String,
+        sa.ForeignKey(
+            projects.c.uuid,
+            name="fk_projects_comments_project_uuid",
+            ondelete="CASCADE",
+            onupdate="CASCADE",
+        ),
+        doc="project reference for this table",
+    ),
```

Comment:
This field will need an index for faster searches, since you use it in pagination queries.

Reply:
yes you are right thanks :+1: